client/src/components/dashboard/DashboardMenuItem.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
+
+// import Activity from './Activity';
+import Schedule from './Schedule';
+import DirectMessages from './DirectMessages';
+import Calendar from './Calendar';
+import Team from './Team';
+import Help from './Help';
+import Profile from './Profile';
+
+export default function DashboardMenuItem({ isOpen, handleClick }) {
+  let { menuItem } = useParams();
+
+  switch (menuItem) {
+    // case 'activity':
+    //   return <Activity handleClick={handleClick} />;
+    case 'schedule':
```

Comment:
Store dashboard (including org) paths as constants or functions to remove hardcoded values

Reply:
fixed in d57efb2311fac1721303ee0d947c10647af5d4af